src/core/DistSparseMatrix.cpp

```diff
@@ -616,6 +616,21 @@ EL_NO_RELEASE_EXCEPT
     return vals_[localInd];
 }
 
+template< typename T>
+T DistSparseMatrix<T>::Get( Int row, Int col) const EL_NO_RELEASE_EXCEPT
+{
+    Int index = Offset( row, col);
+    if( index < 0 || index >= Int(vals_.size())){ return T(0); }
```

Comment:
The behavior of `Graph::Offset( Int row, Int col )` is to call `std::lower_bound` on the sorted indices associated with row `row`, and so the returned index could easily be both greater than 0 and less than `vals_.size()` even though there is no edge at `(row,col)`. Thus, I would suggest changing the check to `if( Source(index) != row || Target(index) != col ) { return T(0); }`